contracts/voter/CommunityVoter.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity =0.8.9;
+
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+
+/// @notice
```

Comment:
worth removing redundant comments from the community deployer?

Reply:
[efc5601](https://github.com/Voltz-Protocol/voltz-core/pull/259/commits/efc56016ecb612382e31a731def95bc55196d393)